publication-event-handlers/src/main/java/no/unit/nva/publication/events/bodies/ScanDatabaseRequest.java

```diff
@@ -0,0 +1,60 @@
+package no.unit.nva.publication.events.bodies;
+
+import static no.unit.nva.publication.events.handlers.PublicationEventsConfig.dynamoImageSerializerRemovingEmptyFields;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.time.Instant;
+import java.util.Map;
+import nva.commons.core.JsonSerializable;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+
+public class ScanDatabaseRequest implements JsonSerializable {
+
+    public static final String START_MARKER = "startMarker";
+    public static final String PAGE_SIZE = "pageSize";
+    @JsonProperty(START_MARKER)
+    private final Map<String, AttributeValue> startMarker;
```

Comment:
A ScanRequest marker as returned by the dynamo client. They will always be consumed only by the client that produced them and therefore there is no reason currently to create a more user-friendly version of them.